L1Trigger/TrackFindingTracklet/interface/PurgeDuplicate.h

```diff
@@ -36,22 +36,24 @@ namespace trklet {
     void execute(std::vector<Track>& outputtracks_, unsigned int iSector);
 
   private:
-    double getPhiRes(Tracklet* curTracklet, const Stub* curStub);
-    bool isSeedingStub(int, int, int);
-    std::string l1tinfo(const L1TStub*, std::string);
-    std::pair<int, int> findLayerDisk(const Stub*);
-    std::vector<double> getInventedCoords(
-        unsigned int,
-        const Stub*,
-        Tracklet*);  // calculate stub coordinates based on tracklet trajectory for prompt tracking
-    std::vector<double> getInventedCoordsExtended(
-        unsigned int,
-        const Stub*,
-        Tracklet*);  // calculate stub coordinates based on tracklet trajectory for extended tracking
-    std::vector<const Stub*> getInventedSeedingStub(
-        unsigned int,
-        Tracklet*,
-        std::vector<const Stub*>);  // return stub with invented x,y,z coords, if it's a seeding one for this tracklet
+    double getPhiRes(Tracklet* curTracklet, const Stub* curStub) const;
```

Comment:
Is there a reason that curTracklet is not also const?

Reply:
I get 
```
 error: passing 'const trklet::Tracklet' as 'this' argument discards qualifiers [-fpermissive]
  588 |     phiproj = curTracklet->proj(curStub->layerdisk()).phiproj();
                                                           ^
``` 
as an error in PurgeDuplicate.cc whenI  make curTracklet const, so I just assumed it wasn't supposed to be made const.

Reply:
OK, a little surprising that accessing the phi projection isn't const but probably beyond the scope of this PR